uuid_arginfo.h

```diff
@@ -42,17 +42,21 @@ ZEND_BEGIN_ARG_WITH_RETURN_TYPE_INFO_EX(arginfo_uuid_variant, 0, 1, IS_LONG, 0)
 ZEND_END_ARG_INFO()
 #endif
 
+#if defined(HAVE_UUID_TIME)
 ZEND_BEGIN_ARG_WITH_RETURN_TYPE_INFO_EX(arginfo_uuid_time, 0, 1, IS_LONG, 0)
 	ZEND_ARG_TYPE_INFO(0, uuid, IS_STRING, 0)
 ZEND_END_ARG_INFO()
+#endif
 
+#if defined(HAVE_UUID_MAC)
 ZEND_BEGIN_ARG_WITH_RETURN_TYPE_INFO_EX(arginfo_uuid_mac, 0, 1, IS_STRING, 0)
 	ZEND_ARG_TYPE_INFO(0, uuid, IS_STRING, 0)
 ZEND_END_ARG_INFO()
 
 #define arginfo_uuid_parse arginfo_uuid_mac
 
 #define arginfo_uuid_unparse arginfo_uuid_mac
+#endif
```

Comment:
Bad position.

Please don't change the arginfo.h files, but generate them  (automatically when stub.php is altered, with PHP >= 8)

Reply:
It's not clear to me how to do that. Do I run the `stub.php`? `phpize`?

Reply:
When you change the stub, a simple make will regenerate the arginfo.

BTW this case can be a bit ugly... perhaps need to move conditional methods at the end to ensure all arginfo are there.